tmd/Topology/analysis.py

```diff
@@ -201,8 +189,12 @@ def barcode_bin_centers(ph, num_bins=100, min_bin=None, max_bin=None):
     bins = np.linspace(min_bin, max_bin, num_bins, dtype=float)
     bin_centers = 0.5 * (bins[:-1] + bins[1:])
 
-    data = [bin_centers[i] for p in ph_2D for i in range(num_bins - 1)
-            if np.max(p) > bins[i + 1] and np.min(p) < bins[i]]
+    data = [
+        bin_centers[i]
+        for p in ph_2D
+        for i in range(num_bins - 1)
+        if np.max(p) > bins[i + 1] and np.min(p) < bins[i]
+    ]
     # data = []
```

Comment:
is this useful?

Reply:
I mean the commented code starting from this line

Reply:
Obviously not :smile: 
But I was not sure why it was there so I kept it. I will add a TODO to mark it. Or maybe @lidakanari thinks we can just remove it?

Reply:
the 206? No, you can remove it :)

Reply:
Ok, thanks!